### PR TITLE
LPD-42282, LPD-39431, LPD-39429 | Create a util to set Timeline info & add timeline info for bookmarks and tags

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-42282
https://liferay.atlassian.net/browse/LPD-39431
https://liferay.atlassian.net/browse/LPD-39429

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:publications

**Comment by brookedalton** [comment]
ci:test:relevant

**Comment by brookedalton** [comment]
ci:report:73749155

**Comment by brookedalton** [comment]
Failures unrelated

File: modules/apps/asset/asset-tags-admin-web/src/main/java/com/liferay/asset/tags/admin/web/internal/portlet/AssetTagsAdminPortlet.java [hunk 1]
```diff
@@ -101,6 +102,9 @@ public void editTag(
 			// Update tag
 
 			_assetTagService.updateTag(null, tagId, name, serviceContext);
+
+			CTCollectionTimelineUtil.setCTTimelineKeys(
```
Comment (david-truong) [inline modules/apps/asset/asset-tags-admin-web/src/main/java/com/liferay/asset/tags/admin/web/internal/portlet/AssetTagsAdminPortlet.java:106]: This is a processAction.  Isn't it too late to set it here?  at this point the user is saving to the database.

File: modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/history/util/CTCollectionTimelineUtil.java [hunk 2]
```diff
@@ -17,6 +17,13 @@
  */
 public class CTCollectionTimelineUtil {
 
+	public static void setClassName(
+		HttpServletRequest httpServletRequest, Class<?> clazz) {
```
Comment (david-truong) [inline modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/history/util/CTCollectionTimelineUtil.java:21]: the renderRequest method should be calling the httpServletRequest one incase the logic ever gets more complicated

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:relevant

**Comment by brookedalton** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:forward:force

**Pull request closed without merging**